### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by joehoyle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,120 +1,65 @@
-Node Tachyon
-============
+<table width="100%">
+	<tr>
+		<td align="left" colspan="2">
+			<strong>Tachyon</strong><br />
+			Faster than light image resizing service that runs on AWS. It's super simple to set up, highly available and very performant.
```
Comment (rmccue) [inline README.md]: Drop the "it's"

File: README.md [hunk 2]
```diff
@@ -1,120 +1,65 @@
...
+## Credits
+Created by Human Made for high volume and large-scale sites, such as [Happytables](http://happytables.com/). We run Tachyon on sites with millions of monthly page views, and thousands of sites.
 
-With the Tachyon plugin installed and active on your local environment add this to your `wp-config-local.php` file:
+Written and maintained by [Joe Hoyle](https://github.com/joehoyle).
 
-```php
-define( 'TACHYON_URL', 'http://localhost:8080/<bucket name>/uploads' );
-```
+Tachyon is inspired by Photon by Automattic Inc. As Tachyon is not an all-purpose image resizer, rather it uses a media library in Amazon S3, it has a different use case to Photon.
```
Comment (rmccue) [inline README.md]: Should link to Photon here, and drop the "Inc".

File: README.md [hunk 3]
```diff
@@ -1,120 +1,65 @@
...
 
-**NOTE:** Currently the region is hardcoded so you don't need the `hmn-uploads-eu-central` part
+Tachyon uses the [Sharp](https://github.com/lovell/sharp) Node.js library for the resizing operations, which in turn uses the great libvips library.
```
Comment (rmccue) [inline README.md]: Mention license here.

File: README.md [hunk 4]
```diff
@@ -1,120 +1,65 @@
...
-```
+| URL Arg | Type | Description |
+|---|----|---|
+|`w`|Number|Max width of the image.|
+|`h`|Number|Max height of the image.|
+|`quality`|Number, 0-100|Image quality.|
+|`crop`|Boolean or "100,50,20,100"|Whether to crop the image, or resize proportionally.|
 
-With no options passed you should see the server running by default on http://localhost:8080
+For more details checkout the [docs](https://engineering.hmn.md/projects/tachyon/).
```
Comment (rmccue) [inline README.md]: This link should be more promenant. Maybe add to the table up the top, or if not, add immediately after the table.

File: README.md [hunk 5]
```diff
@@ -1,120 +1,65 @@
...
+You'll need to upload the latest release to Amazon S3 and put the location in the to the CloudFormation stack configuration.
 
-The `cloudformation-template.json` in the root should be used to instantiate a new Stack in CloudFormation. You'll need to build and zip the Lambda function (see below) and place the zip on S3. Pass this ZIP location in the CloudFormation template parameters.
+## Using
 
-## Configuration
+Tachyon provides a simple HTTP interface in the form of:
 
-Populate the `config.json` with the AWS region and bucket name you want to us. In the following format:
+https://{tachyon-domain}/my/image/path/on/s3.png?w=100&h=80
```
Comment (rmccue) [inline README.md]: Should be marked as code

File: README.md (hunk 4 above)
Comment (joehoyle) [inline README.md]: I was thinkign linking in the Github description too

Comment (rmccue) [inline README.md]: Sure, both is fine.

File: README.md [hunk 6]
```diff
@@ -1,120 +1,65 @@
...
+- Expects original image files to be stored on Amazon S3.
+- Only supports simple image resizing, not a full image manipulation service.
 
-If you're running on OSX make sure you have libvips installed, the easiest way is to use homebrew:
+Tachyon works best with WordPress, coupled with [S3 Uploads](github.com/humanmade/s3-uploads) and the [Tachyon Plugin](https://github.com/humanmade/tachyon-plugin).
 
-```
-brew install homebrew/science/vips --with-webp --with-graphicsmagick
-```
+![](http://joehoyle-captured.s3.amazonaws.com/I77brfBv.png)
```
Comment (rmccue) [inline README.md]: Before I merge this, will add this to the Engineering Handbook so it has a more permanent URL

Comment (rmccue) [inline README.md]: URL is https://engineering.hmn.md/projects/tachyon/diagram.png

**Pull request merged**